File: Selenium/Activities/Activity_6.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity_6 {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new ChromeDriver();
        Actions act = new Actions(driver);
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("The title of WebPage is: " + driver.getTitle());
        act.keyDown(Keys.SHIFT).sendKeys("s").build().perform();
        Thread.sleep(1000);
        act.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).build().perform();
        driver.quit();
    }
}

File: Selenium/Activities/Activity_6.py
import time

from selenium import webdriver
from selenium.webdriver import Keys


with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("The page title is: ", driver.title)
    act = webdriver.ActionChains(driver)
    act.key_down(Keys.SHIFT).send_keys("s").key_up(Keys.SHIFT).perform()
    time.sleep(5)
    act.key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()
    driver.quit()